Gemfile

```diff
@@ -20,6 +20,7 @@ gem 'rails-i18n'
 gem 'rails_safe_tasks', '~> 1.0'
 
 gem "activerecord-import"
+gem "db2fog", github: "Matt-Yorkley/db2fog", branch: "rails-5"
```

Comment:
I think it's a much better idea to fork in under openfoodfoundation's GH organization. As is, none of the other devs will have permission to touch anything on that branch if the need arises. I've had bad experiences in the past with this same issue.

Reply:
A yeah, I was just doing some quick tests with this to check it worked. Manually tested and it's working fine, I'll move it to OFN :+1: